client/src/containers/pages/Home/Home.tsx

```diff
@@ -30,15 +30,15 @@ const Home: React.FC<HomeProps> = (props: HomeProps) => {
         {roles && roles.includes('ROLE_EDIT_KEYCLOAK_USERS') && (
           <Col className="gutter-row" span={12}>
             <Link to={URL_ADMIN} className="admin-link">
-              <Button color="outline" className="btn-links">
+              <Button color="outline" className="home btn-links">
```

Comment:
should home be applied here, seems like it contextually refers to something else.

Reply:
Thse are the home buttons responsible for styling the Home component large buttons I added home class. what name would be best to target these buttons ?

Reply:
cool, thought home here was referring to the whole page view, in that sense it felt like it should have been hosted to the div element encompassing the home page.